roles/mappingservice/templates/system-master-access-token-secretdefinition.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: secrets-manager.tuenti.io/v1alpha1
+kind: SecretDefinition
+metadata:
+  name: system-master-access-token
```

Comment:
Is this secret required on other workloads (by the resource name it seems it could be a possibility)?

Because what we previously agreed, was to add the prefix name of each workload to the created `SecretDefinition` and `Secret` object names (an so changing the deployment mount), so although maybe the vault secret path is used on different workloads, each workload manage its own lifecycle objects (easily identified by the prefix), to avoid possible race conditions of owner references...

For example, there is a secret used by System and Backend calle `system-events-hook`, but Backend workload is creating its own secretdef/secret  `backend-system-events-hook` https://github.com/3scale/saas-operator/blob/master/roles/backend/templates/backend-system-events-hook-secretdefinition.yaml#L4

In the CR examples you provide, it seems you already have something similar on vault     `systemAdminTokenVaultPath: secret/data/openshift/stg-saas-ocp/3scale/mappingservice-system-master-access-token`

Reply:
Yep that's right, this secret is only used by mapping-service. So should we rename it to `mappingservice-system-master-access-token`?

Reply:
I've seen it is already implemented, thanks!